### PR TITLE
Add a "mozilla" tag to each and every article.

### PR DESCRIPTION
**Pull request by almet**
Useful to be integrated to the planet, see
https://bugzilla.mozilla.org/show_bug.cgi?id=1156306

### CODE REVIEW

File: add_mozilla_tag.py [hunk 1]
```diff
@@ -0,0 +1,11 @@
+from pelican import signals
+
+def add_category(generator):
+    for article in generator.articles:
+        if hasattr(article, 'tags'):
+            article.tags.append('mozilla')
+        else:
+            setattr(article, 'tags', ['mozilla'])
```
Comment (almet) [inline add_mozilla_tag.py:8]: Yeah, I know, this is due to [how pelican handles metadata](https://github.com/getpelican/pelican/blob/master/pelican/contents.py#L62-L65) (and the worse here is that **I** wrote this code…)

**Comment by leplatrem** [comment]
I'm not against it :) r+

Even though I wonder how we would write an article that is not tagged `mozilla` with this plugin

**Comment by almet** [comment]
We'll figure if we need to (adding some metadata to the article and a conditional in the plugin).

**Pull request merged**